packages/ember-application/lib/system/resolver.js

```diff
@@ -254,13 +253,12 @@ export default EmberObject.extend({
   */
   lookupDescription(fullName) {
     let parsedName = this.parseName(fullName);
-    let description;
 
     if (parsedName.type === 'template') {
       return `template at ${parsedName.fullNameWithoutType.replace(/\./g, '/')}`;
     }
 
-    description = `${parsedName.root}.${StringUtils.classify(parsedName.name).replace(/\./g, '')}`;
+    let description = `${parsedName.root}.${StringUtils.classify(parsedName.name).replace(/\./g, '')}`;
```

Comment:
Isn't this hoisted to the top of the file, incurring the cost of transforming the description? I think that's why the original was split up.